### PR TITLE
Remove tags field from RRD hosts.

### PR DESCRIPTION
**Pull request by vkalintiris**
##### Summary

My understanding is that this field has been
deprecated in favour of RRD labels.

##### Test Plan

- CI jobs.

### CODE REVIEW

File: database/sqlite/sqlite_metadata.c [hunk 1]
```diff
@@ -354,7 +354,8 @@ static int store_host_metadata(RRDHOST *host)
     if (unlikely(rc != SQLITE_OK))
         goto bind_fail;
 
-    rc = bind_text_null(res, ++param, rrdhost_tags(host), 1);
+    // TODO: remove/migrate
```
Comment (vkalintiris) [inline database/sqlite/sqlite_metadata.c:357]: @stelfrag Okay for you to remove this whenever the need arises to perform the next migration? I'm not familiar with the code/component that performs the migrations and considering the simplicity of the PR, I didn't want to add more complexity.

**Comment by thiagoftsm** [comment]
Hello @vkalintiris ,

The `unittest` is affected by this PR, please, take a look.

Best regards!

**Comment by thiagoftsm** [comment]
The PR is also not compiling locally with the following error:

```sh
   51 | #define HAVE_BACKTRACE
      | 
<command-line>: note: this is the location of the previous definition
/home/thiago/Netdata/tests_netdata/src/streaming/receiver.c: In function 'rrdpush_receive':
/home/thiago/Netdata/tests_netdata/src/streaming/receiver.c:630:85: error: 'struct receiver_state' has no member named 'tags'
  630 |     (void)appconfig_set_default(&stream_config, rpt->machine_guid, "host tags", (rpt->tags)?rpt->tags:"");
      |                                                                                     ^~
/home/thiago/Netdata/tests_netdata/src/streaming/receiver.c:630:96: error: 'struct receiver_state' has no member named 'tags'
  630 |     (void)appconfig_set_default(&stream_config, rpt->machine_guid, "host tags", (rpt->tags)?rpt->tags:"");

```

Best regards!

**Review by thiagoftsm — APPROVED** [review]
After last commits, I could compile, and the PR is working as expected. LGTM!

**Pull request merged**